history.md

```diff
@@ -0,0 +1,3 @@
+Currently, every backup that is taken is called exactly the same. The outcome of this is that the newest backup always overwrites the previous one. ReaR, however, allows the usage of the configuration parameter ISO_PREFIX to change the name of the generated ISO. 
+
+However, there are a set of other techniques that can be applied, jbadiapa is working on it, trying different capabilities.
```

Comment:
IMHO this last thing should be deleted.

Reply:
That comment is there so you write a much more detailed spec with the solution you are applying to run the history. That's why this spec is almost empty: as there are different ways of configuring ReaR to achieve this, I did not want to propose something different that what you were implementing.